node/resolvers/Queries/Users.ts

```diff
@@ -816,6 +816,44 @@ export const getOrganizationsByEmail = async (
   }
 }
 
+export const getOrganizationsByEmailPaginated = async (
+  _: any,
+  {
+    email = '',
+    page = 1,
+    pageSize = 25,
+  }: {
+    email: string
+    page: number
+    pageSize: number
+  },
+  ctx: Context
+) => {
+  const {
+    clients: { masterdata },
+    vtex: { logger },
+  } = ctx
+
+  try {
+    const data = await masterdata.searchDocumentsWithPaginationInfo({
+      dataEntity: config.name,
+      fields: ['clId', 'costId', 'id', 'orgId', 'roleId'],
+      pagination: { page, pageSize },
```

Comment:
when querying paginated data from Master Data, we should add `sort` (e.g. `sort: 'id asc',` otherwise we might get inconsistent data result (e.g. duplicated entries) when querying a lot of data/pages.